common-docs/reference/math/random-range.md

```diff
@@ -0,0 +1,15 @@
+# Random Range
+
+Returns a pseudo-random number in the range ``[min, max];`` that is, from ``0`` (inclusive) up to including ``1`` (inclusive), which you can then scale to your desired range. 
```

Comment:
I was trying to map scratch: https://wiki.scratch.mit.edu/wiki/Pick_Random_()_to_()_(block)